tests/test_subprocess.py

```diff
@@ -44,7 +44,7 @@ def test_please_wait(self):
         r = requests.get(self.url)
         self.assertIn('Please wait', r.content)
 
-    def test_container(self):
+    def test_container(self, mock_logger):
```

Comment:
I approved this pr, but just saw the failing build. Removing  the `mock_logger` arg should do the trick